javascript/src/components/dashboard/EditDashboardModal.jsx

```diff
@@ -1,90 +1,103 @@
-'use strict';
+const $ = require('jquery');
 
-var $ = require('jquery');
+const React = require('react');
+const BootstrapModal = require('../bootstrap/BootstrapModal');
+const DashboardStore = require('../../stores/dashboard/DashboardStore');
 
-var React = require('react');
-var BootstrapModal = require('../bootstrap/BootstrapModal');
-var DashboardStore = require('../../stores/dashboard/DashboardStore');
+const EditDashboardModal = React.createClass({
+  getInitialState() {
+    return {
+      id: this.props.id,
+      description: this.props.description,
+      title: this.props.title,
+    };
+  },
+  getDefaultProps() {
+    return {
+      action: 'create',
+    };
+  },
+  render() {
+    const header = (
+      <h2 className="modal-title">{this._isCreateModal() ? 'New Dashboard' : 'Edit Dashboard ' + this.props.title}</h2>
+    );
+    const body = (
+      <fieldset>
+        <div className="form-group">
+          <label htmlFor={`${this.props.id}-title`}>Title:</label>
+          <input id={`${this.props.id}-title`}
```

Comment:
Why don't we use the react-bootstrap components for this? Would make the code much cleaner imo.